src/middlewared/middlewared/main.py

```diff
@@ -192,32 +192,14 @@ async def __crash_reporting(self, exc_info):
                 )
 
     async def subscribe(self, ident, name):
-
         if ':' in name:
             shortname, arg = name.split(':', 1)
         else:
             shortname = name
             arg = None
-        event_source = self.middleware.get_event_source(shortname)
-        if event_source:
-            for v in self.__event_sources.values():
-                # Do not allow an event source to be subscribed again
-                if v['name'] == name:
```

Comment:
Would we now allow users to subscribe to same event source multiple times ?

Reply:
Yes, as we allow to subscribe to the same event multiple times.